<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroWindow</class>
 <widget class="QMainWindow" name="IntroWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>키오스크</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/icons/arrow.png</normaloff>src/images/icons/arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>560</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>원하시는 채소를 모두 선택하세요.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>10</height>
     </rect>
    </property>
    <property name="value">
     <number>80</number>
    </property>
   </widget>
   <widget class="QPushButton" name="LettuceButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>600</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>양상추</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/vegetable/lettuce.png</normaloff>src/images/assets/vegetable/lettuce.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AvocadoButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>600</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>아보카도</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/vegetable/avocado.png</normaloff>src/images/assets/vegetable/avocado.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="OliveButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>600</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>올리브</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/vegetable/olive.png</normaloff>src/images/assets/vegetable/olive.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="TomatoButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>600</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>토마토</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/vegetable/tomato.png</normaloff>src/images/assets/vegetable/tomato.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="OnionButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>600</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>적양파</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/assets/vegetable/onion.png</normaloff>src/images/assets/vegetable/onion.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectCompleteButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>780</y>
      <width>600</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>선택 완료</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>src/images/icons/check.png</normaloff>src/images/icons/check.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
